### PR TITLE
adding sleep() for connection time out, more clear inputs to update_worksheet(), deleting empty line on 178 in inout.py

### PR DESCRIPTION
**Pull request by theanhgen**

### CODE REVIEW

File: ups_modul/inout/inout.py [hunk 1]
```diff
@@ -171,11 +170,12 @@ def loop():
 # weird stuff
 if __name__ == "__main__":
     setup_inout()
-    connected = False
-    while connected == False:
+    connected = "OFF"
```
Comment (nguyeho7) [inline ups_modul/inout/inout.py]: comparing strings is O(n) time. Consider using boolean or enum instead.

File: ups_modul/inout/inout.py [hunk 2]
```diff
@@ -171,11 +170,12 @@ def loop():
...
+    while connected == "OFF":
         try:
             check_new_users()
-            connected = True
+            connected = "ON"
         except:
-            sleep(100)
+            time.sleep(100)
+        print(connected)
```
Comment (nguyeho7) [inline ups_modul/inout/inout.py]: Do we need this print function?

**Review by nguyeho7 — CHANGES_REQUESTED** [review]

File: ups_modul/inout/inout.py (hunk 2 above)
Comment (theanhgen) [reply ups_modul/inout/inout.py]: just for checking whether the script is on or not, handy when debuging

File: ups_modul/inout/inout.py (hunk 1 above)
Comment (theanhgen) [reply ups_modul/inout/inout.py]: changed back to boolean and adding print when connection is up

**Review by nguyeho7 — APPROVED** [review]

**Pull request merged**